### DISCUSSION TITLE
Stopped working on Opera on Mac OS 13.2

### DISCUSSION THREAD

**Discussion by aMittalAnshul** [Acidham/chromium-hist-bookmarks#29] [General]
Hello, I've just upgraded my Mac to Ventura and now this extension is no longer working with Opera.  Can someone advise what got broken?  

Here is my setup with Alfred 5.  

<img width="1336" alt="Screenshot 2023-05-02 at 10 07 22 AM" src="https://user-images.githubusercontent.com/17104101/235726084-fa6d4610-bd4e-4564-bfea-d836543f1f0e.png">

<img width="1336" alt="Screenshot 2023-05-02 at 10 07 14 AM" src="https://user-images.githubusercontent.com/17104101/235726166-52e25833-afe3-4ec8-a8e2-5c22a8ad7bbc.png">

**Comment by Acidham** [comment]
What is the error when you execute it? Click on the little bug top right and execute the workflow.

BTW: I am on Ventura 13.3.1, and it works with Opera

**Comment by aMittalAnshul** [comment]
Here is a screencast of the usage.  

https://user-images.githubusercontent.com/17104101/235728087-8ef10f5d-fdaa-4903-9241-3dcfedf2ed80.mp4

**Comment by aMittalAnshul** [comment]
Here is the debug.  

[10:33:37.406] Chromium Bookmarks and History Search[Script Filter] Queuing argument 'hello'
[10:33:37.532] Chromium Bookmarks and History Search[Script Filter] Script with argv 'hello' finished
[10:33:37.538] ERROR: Chromium Bookmarks and History Search[Script Filter] Code 1: xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

---
So it sounds like I need to install command line dev tools?

**Comment by Acidham** [comment]
Ok I see. The error comes from Command Line Tools. For more info, see here: https://osxdaily.com/2021/07/19/fix-xcrun-error-invalid-active-developer-path-error-terminal-mac/

As described on the page, execute `xcode-select --install` in the terminal and try again.

**Reply by aMittalAnshul** [reply]
Ahh, I will do that.  Thanks a bunch for the quick reply here.  Love the extension 👍

**Reply by aMittalAnshul** [reply]
It's working now.  Thank you.
